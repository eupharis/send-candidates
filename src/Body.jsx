import React from 'react';

const bodyStyle = {
  width: '100%',
  height: '480px',
  marginBottom: '20px',
};

function Body(props) {
  return <textarea style={bodyStyle} onChange={props.onChange} value={props.body} />;
}

export default Body;
