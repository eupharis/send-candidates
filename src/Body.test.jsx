import React from 'react';
import renderer from 'react-test-renderer';
import Body from './Body';

test('Link changes the class when hovered', () => {
  const component = renderer.create(<Body body="foo" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
