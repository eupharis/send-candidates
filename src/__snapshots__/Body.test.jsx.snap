exports[`test Link changes the class when hovered 1`] = `
<textarea
  onChange={undefined}
  style={
    Object {
      "height": "480px",
      "marginBottom": "20px",
      "width": "100%",
    }
  }
  value="foo" />
`;
